src/base/Nebi.js

```diff
@@ -49,10 +49,35 @@ class Nebi extends Client {
       restTimeOffset: 0,
     });
 
-    this.commands = new Collection();
+    /**
+		 * Collection para eventos registrados
+		 * @type {Collection}
+		*/
     this.events = new Collection();
+
+    /**
+		 * Collection para as interações registradas
+		 * @type {Collection}
+     * @type {Collection}
+     * @type {Collection}
+     * @type {Collection}
+		*/
+    this.commands = new Collection();
     this.buttons = new Collection();
     this.selects = new Collection();
     this.modals = new Collection();
+
+    /**
+		 * Permitir acesso ao mongo
+		 * @type {function}
+		*/
+		this.mongoose = require('../database/mongoose');
```

Comment:
Por que isso está indentado assim?

Reply:
Atualizações futuras, vou tava atualizando para facilitar. Praticamente se eu coloco `this.mongoose` numa classe que extende a classe Client, toda vez que eu chamar o `client.<var>` irá ter a opção mongoose. Praticamente tudo no discord.js tem esse client, então deixaria mais fácil do que ficar usando require. Eu vi isso num outro código e parando para pensar, usar o require só uma vez talvez interfira em alguma otimização do que chamar várias vezes.

Reply:
A questão foi mais de linting, esses dois tabs saem do padrão de estilo do código por nenhum motivo aparente.

Reply:
É que tive uma epifania que comecei a querer alterar todos os códigos, mas no fim cansei. Vou deixar para essas mudanças pra quando chegar a nova versão, no fim, vou acabar tirando isso

Reply:
Eu tirei a bomba, vou colocar quando formos refazer tudo